### ISSUE TITLE
Basic Questions Regarding Yolov5 and need confirmation from @glenn-jocher

### ISSUE THREAD

**Issue by akshat-suwalka** [ultralytics/yolov5#3832]
## ❔Question

Simply **using all the default setting of YOLOv5s**

1. What are the loss functions used for box, objectness, classification?
Ans => What I found on searching over internet that :-
 CIOU Loss is used as the loss of bounding box in yolov5 and IoU is used for NMS
 Binary Cross-Entropy with Logits Loss function from PyTorch for loss calculation of class probability and object score.

Is the above answer fully correct for both training and testing case, Please if there is any correction let me know??

2. Is precision and recall defined for detection or both detection and
classification?
Ans => I am not sure @glenn-jocher Can you please enlighten me?

3. You have reported values for precision and recall. What are the
values of recognition accuracy?
Ans => Any comments?

4. How is mean average precision computed? How do you get the P@0.95 values?
Ans => I know it but if you can answer then it will increase my understanding more.... Can you please answer?

5. Was cross validation used?
Ans => Not sure....any comments @glenn-jocher ?

6. Can you please tell computational time results like using default setting how much time the YoloV5small is taking...?

**Comment by glenn-jocher** [comment]
@akshat-suwalka for details on loss function see loss.py:
https://github.com/ultralytics/yolov5/blob/master/utils/loss.py

For details on metrics see metrics.py:
https://github.com/ultralytics/yolov5/blob/master/utils/metrics.py

Speeds are clearly displayed in README.
